pom.xml

```diff
@@ -514,7 +520,7 @@
                         <dependency>
                             <groupId>com.puppycrawl.tools</groupId>
                             <artifactId>checkstyle</artifactId>
-                            <version>8.18</version>
+                            <version>8.45.1</version>
```

Comment:
Why don't we use latest version ? [10.3.2](https://mvnrepository.com/artifact/com.puppycrawl.tools/checkstyle/10.3.2)

Reply:
Because it breaks due to the current required version of java for that higher version. In other words, we need to upgrade java first, and then the plug-in version.